src/ZwiftPacketMonitor.csproj

```diff
@@ -1,26 +1,30 @@
-<Project Sdk="Microsoft.NET.Sdk" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+﻿<Project Sdk="Microsoft.NET.Sdk" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>net5.0</TargetFramework> <!-- netstandard2.0 -->
     <id>ZwiftPacketMonitor</id>
-    <Version>1.2.8</Version>
+    <Version>1.2.9.1</Version>
     <GeneratePackageOnBuild Condition="'$(Configuration)' == 'Release'">true</GeneratePackageOnBuild>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
-    <Authors>Brad Walker (@braddwalker)</Authors>
+    <Authors>ruffk / Brad Walker (@braddwalker)</Authors>
```

Comment:
Not to pick nits here, but let's just switch the order of names, for obvi reasons.

Reply:
Sorry for injecting myself!  To explain, I've been trying to reach you via email for several weeks.  I had some Mac dev questions and also needed your assistance with this fix.  As I needed to release a new version of ZwiftActivityMonitor I needed to get the changes done so I updated my fork and published my own package.  I have removed all references and will switch back to your package when published.  I will also publicly hide mine.  So glad you're okay, I was really concerned!

Reply:
omg you're right, I totally missed your email -- sorry! It's been a super busy spring/summer. I've starred your email address so that won't happen again :)